backend/src/main/java/com/zeros/backend/RestaurantResource.java

```diff
@@ -0,0 +1,21 @@
+package com.zeros.backend;
+
```

Comment:
Why all the classes are in the same package ? isn't it wiser to put resources classes for instance in a different package ?

Reply:
to go faster i didn't regroupe those class in packages;
i hope to refactor this in future